data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/configuration/EngineType.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.source.rds.configuration;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public enum EngineType {
+
+    MYSQL("mysql"),
+    AURORA_MYSQL("aurora-mysql");
```

Comment:
Do we need to know the difference between Aurora and non-Aurora?

If so, would it make more sense to have Aurora as a boolean in the configuration?

```
engine: mysql
aurora: true
```

Reply:
Yes, there are some binlog related settings that are unique to Aurora, there could be other differences. Will make the change.